tutorials/spectral-cube-reprojection/SpectralCubeReprojectExample.ipynb

```diff
@@ -0,0 +1,635 @@
+{
```

Comment:
<p>"We can do the spatial or spectral step first" --&gt; "Now we have to decide which to do first: combine the two data cubes onto the same spectral grid or the same spatial grid."</p><p> </p><p>something like that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/504/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
revised